packages/react-devtools-extensions/README.md

```diff
@@ -7,15 +7,28 @@ The easiest way to install this extension is as a browser add-on:
 * [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
 
 ## Development
-
 You can also build and install this extension from source.
 
-DevTools embeds local versions of several NPM packages also in this workspace. If you have not already built them, you'll need to do that before getting started by running the following command in the root directory of this repository:
+### Prerequisite steps
+DevTools depends on local versions of several NPM packages<sup>1</sup> also in this workspace. You'll need to either build or download those packages first.
+
+<sup>1</sup> Note that at this time, an _experimental_ build is required because DevTools depends on the `createRoot` API.
+
+#### Build from source
+To build dependencies from source, run the following command from the root of the repository:
 ```sh
-yarn build -- react,react-dom,react-is,scheduler --type=NODE
+RELEASE_CHANNEL=experimental \
+  yarn build \
+    -- react/index,react-dom,react-is,react-debug-tools,scheduler \
+    --type=NODE
 ```
-
-Once the above packages have been built, you can build the extension by running:
+#### Download from CI
+To use the latest build from CI, run the following command from the root of the repository:
```

Comment:
It’s not clear that this is an alternative to the previous step.

Reply:
Why not? They're subheaders under the same header.

Reply:
I could make the header size smaller maybe?

Reply:
Is there a problem with just having one of them? Like, downloading from CI could be the suggested option, and you have a postscript that mentions that’s you can also build from source if needed. 

I’m thinking from perspective of someone following instructions to build the extension, who doesn’t understand what these deps are, or what context they provide. If *I* had to quickly make a build and followed these instructions, I might have done both steps. Ofc this is just a nit, so feel free to ignore.

Reply:
Yes. It's not sufficient.

*We* probably want to build from source, so we can test both DevTools *and* React changes.

External contributors (or Mozilla reviewers) probably want to download a CI build because it's faster and their purpose is not to test React.